proc_scripts/phasing.py

```diff
@@ -304,15 +304,19 @@ def hermitian_function_test(s, down_from_max=0.5, rel_shift=1.5,shift_points=120
         fig, (ax1,ax2,ax3) = plt.subplots(3,1)
         fl.next('mirror tests', fig=fig)
         def real_imag_mirror(forplot, ax):
-            l = fl.plot(forplot.real, alpha=0.5, ax=ax, label='real')
-            fl.plot(forplot.C.setaxis('t2', lambda x: -x), color=l[-1].get_color(),alpha=0.2,ax=ax)
+            l = fl.plot(forplot.real, alpha=0.5, ax=ax, label='real',
+                    human_units=False)
```

Comment:
when is a proper time to use human_units=False vs not having that addition? I know there are times where we do and don't want this but what are those times?

Reply:
we shouldn't really have to use them ever -- it's a bit of a bug that hasn't been fixed.  As is, the only reasonable time is, like here, when you have two sets of axis labels that are very different sizes -- so that it decides the units it would apply to each are different